### PR TITLE
Updated Event payload and added comments

### PR DESCRIPTION
**Pull request by bobbytreed**
Issue Comments are a separate event.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ async function main() {
 
 		// create right patch document depending on the action tied to the issue
 		// update the work item
+		console.log(`Action: ${vm.action}`)
```
Comment (cephalin) [inline index.js:73]: do you mean for this debug code to remain?

File: index.js [hunk 2]
```diff
@@ -317,6 +318,7 @@ async function update(vm, workItem) {
 async function comment(vm, workItem) {
 	let patchDocument = [];
 
+	console.log(`CommentText: ${vm.comment_text}`)
```
Comment (cephalin) [inline index.js:321]: ditto

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -3,6 +3,8 @@ name: Sync Issue to Azure DevOps work item
 on:
   issues:
     types: [opened, edited, deleted, closed, reopened, labeled, unlabeled, assigned, unassigned]
+  issue_comment:
+    types: [created, edited]
```
Comment (cephalin) [inline .github/workflows/main.yml:7]: nice is that all that's required to turn on comments?

File: index.js (hunk 1 above)
Comment (bobbytreed) [reply index.js:73]: Yes. I want to make sure that we are seeing all actions coming in and that we have a case for them.

File: index.js (hunk 2 above)
Comment (bobbytreed) [reply index.js:321]: Yeah, until we know we have comments synced.

File: .github/workflows/main.yml (hunk 3 above)
Comment (bobbytreed) [reply .github/workflows/main.yml:7]: Yeah, I did some research and I was wondering where he was getting "created" from. It's basically just GraphQL syntax

**Pull request merged**